wolfcrypt/src/port/intel/quickassist.c

```diff
@@ -557,6 +557,12 @@ int IntelQaOpen(WC_ASYNC_DEV* dev, int devId)
         return ASYNC_INIT_E;
     }
 
+    if (devId >= g_numInstances) {
+        fprintf(stderr, "devId %d out of range. Number of instances=%u\n",
```

Comment:
Please use the specific text I outlined. Thanks

Reply:
Done